mesonbuild/compilers/rust.py

```diff
@@ -355,5 +355,8 @@ def get_dependency_gen_args(self, outtarget: str, outfile: str) -> T.List[str]:
     def get_output_args(self, outputname: str) -> T.List[str]:
         return []
 
+    def get_exe(self) -> str:
+        return self.exelist[0]
```

Comment:
This should go in the base Compiler class. There's a lot of places that we use this from C and C++